native/effektio/Cargo.toml

```diff
@@ -6,13 +6,13 @@ rust-version = "1.58"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
-
 [build-dependencies]
-ffi-gen = { git = "https://github.com/effektio/ffi-gen" }
+ffi-gen = { git = "https://github.com/bmcpt/ffi-gen", rev = "fb2e59725401908539525e890c5961d3f6824094" }
 
 [dependencies]
 anyhow = "1.0.51"
-ffi-gen-macro = { git = "https://github.com/effektio/ffi-gen" }
```

Comment:
same as above.

Reply:
This PR depends on https://github.com/effektio/ffi-gen/pull/4

Reply:
@bmcpt merged.